rdkcompositor.h

```diff
@@ -74,6 +74,7 @@ namespace RdkShell
             void setVirtualResolution(uint32_t virtualWidth, uint32_t virtualHeight);
             void enableVirtualDisplay(bool enable);
             bool getVirtualDisplayEnabled();
+	    uint32_t mSurfaceCount=0;
```

Comment:
make it as private variable and have a public method getNumberOfSurfaces() and use it for retrieval from compositorcontroller.cpp